.github/CODEOWNERS

```diff
@@ -0,0 +1,17 @@
+# ----------------------------------------------------------------------------
+# MDN bcd-utils CODEOWNERS
+# ----------------------------------------------------------------------------
+# Order is important. The last matching pattern takes precedence.
+# For more detailed information, see:
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+# ----------------------------------------------------------------------------
+
+# ----------------------------------------------------------------------------
+# DEFAULT OWNERS
+# ----------------------------------------------------------------------------
+*    @mdn/core-dev
+
+# These are @mdn-bot because the auto-merge GHA workflow uses the PAT of this account.
+# If another reviewer is specified, update the PAT token or auto-merge will cease to be automatic.
+/package.json      @mdn-bot
+/package-lock.json @mdn-bot
```

Comment:
@mdn-bot doesn't have access to the repo, and we don't seem to have automerge or dependabot set up on this repo, should we remove this bit? It generates an error in the github ui

Reply:
Fixed in [74c86e0](https://github.com/mdn/bcd-utils/pull/27/commits/74c86e0512dcb019964ecd9f5e5b7fb287a45cf1).